daycare/test_daycare.py

```diff
@@ -23,6 +30,16 @@ def create_daycare_user():
                           capacity=CAPACITY, area=AREA, city=CITY, address=ADDRESS)
 
 
+@pytest.fixture
```

Comment:
Can you create a dedicate test file for image?

Reply:
I also think you should separate the two.

Reply:
Done